app/models/resource_request.rb

```diff
@@ -27,15 +33,9 @@ class ResourceRequest < ApplicationRecord
   # enqueue_derivativo_job ends up making requests to Derivativo which makes requests back to
   # Hyacinth and those requests fail when there's an existing lock on the squlite database.
   # See: https://flexport.engineering/how-to-safely-use-activerecords-after-save-efde2b52baa3
-  after_commit :enqueue_derivativo_job, on: :create
+  after_commit :run_create_callback, on: :create
```

Comment:
I feel like the naming of `run_create_callback` is a little too literal, and could be confusing.  If feels similar to creating a method called `after_create_method` and binding it to the model's ActiveRecord `after_create` callback.  Also, I'm concerned that if someone on our team refers to the model's "create callback" that everyone else would assume they were talking about the ActiveRecord callbacks around create.

But I understand what `run_create_callback` is doing, and why you named it this way, and I am also finding it difficult to think of an alternate name.

You've introduced a cool way to dynamically add an after_create (or maybe more accurately, after_commit) hook on an instance of a ResourceRequest.  I'm going to think about this a little more and see if I can come up with an alternate name.